### PR TITLE
Minor fixes.

### PR DESCRIPTION
**Pull request by vng**

### CODE REVIEW

File: drape/vulkan/vulkan_utils.hpp [hunk 1]
```diff
@@ -70,6 +70,12 @@ struct SamplerKey
                                      dp::vulkan::GetVulkanResultString(statusCode))); \
   } while (false)
 
+#define CHECK_VK_CALL_EX(method, msg) \
+  do { \
+    VkResult const statusCode = method; \
+    CHECK(statusCode == VK_SUCCESS, msg); \
```
Comment (biodranik) [inline drape/vulkan/vulkan_utils.hpp:76]: CHECK_EQUAL?

**Review by biodranik — APPROVED** [review]

File: drape_frontend/rule_drawer.cpp [hunk 2]
```diff
@@ -322,7 +322,7 @@ void RuleDrawer::ProcessAreaAndPointStyle(FeatureType & f, Stylist const & s, TI
     apply.ProcessAreaRule(s.m_areaRule, areaDepth, false);
 
   /// @todo Can we put this check in the beginning of this function?
-  if (!IsDiscardCustomFeature(f.GetID()))
+  if (applyPointStyle && !IsDiscardCustomFeature(f.GetID()))
```
Comment (biodranik) [inline drape_frontend/rule_drawer.cpp:325]: 1. [drape] Fixed possible uninitialized center point for area Feature. => It's not "possible", it's definitely uninitialized :)
2. I confirm that this fix works.
3. The code of this function looks strange. Why ApplyAreaFeature is used even if applyPointStyle is true, for example?

File: drape_frontend/gui/layer_render.cpp [hunk 3]
```diff
@@ -185,6 +185,10 @@ class ScaleFpsLabelHandle : public MutableLabelHandle
 
     auto const vs = static_cast<float>(df::VisualParams::Instance().GetVisualScale());
     m2::PointF offset(10.0f * vs, 30.0f * vs);
+#ifdef OMIM_OS_IPHONE
+    // Move below dynamic island.
```
Comment (biodranik) [inline drape_frontend/gui/layer_render.cpp:189]: What is dynamic island?

**Review by biodranik — APPROVED** [review]

File: drape_frontend/gui/layer_render.cpp (hunk 3 above)
Comment (dvdmrtnz) [reply drape_frontend/gui/layer_render.cpp:189]: The iPhone camera cutout in the screen is called Dynamic Island by Apple

File: drape_frontend/rule_drawer.cpp (hunk 2 above)
Comment (vng) [reply drape_frontend/rule_drawer.cpp:325]: Area Feature process area (hatching, filling) and point (symbol, text) styles.

**Review by biodranik — APPROVED** [review]
Merge?

**Pull request merged**